frontend/package-lock.json

```diff
@@ -1099,7 +1099,7 @@
       "requires": {
```

Comment:
Looks like some package-lock changes still made it in. Maybe we should reset these?

Reply:
I also think a number of these changes made it in separately, so maybe reset these changes and then pull from staging?